ci/assets/terraform/template.tf

```diff
@@ -1,12 +1,8 @@
-variable "access_key" {}
```

Comment:
@keymon Won't this rely on default environment variables being set on the machine running the terraform template?

Reply:
Yes, exactly. 

This specific change is isolated and explained in this commit: https://github.com/cloudfoundry-incubator/bosh-aws-cpi-release/pull/70/commits/2adce9a7062a0cf26959ea08ff4d047fe7472ca3 , have a look at it. It can be dropped if needed, ofc.

The thing is that when we are calling terraform, we are explicitly passing the standard variables from the environment, which is what the terraform provider does by itself.

Reply:
@keymon Sorry, I didn't read the commit message. We wanted to be explicit when setting key id and secret key to avoid a destructive operation in a production environment.
That being said it would be better to revert that change.

Reply:
I do not have problem to revert it, but I want to first challenge the justification if you don't mind. I am not sure if you are achieving what you want.

The original implementation in master would always pass the credentials in the environment. If in the environment there is a prod environment, it will do so. The only thing it prevents is not use the profiles file.

But I think there is a better way to secure the environment:

 1. Add  `shared_credentials_file = "/dev/null"` to disable the usage of the profiles file.
 2. Add a `allowed_account_ids = [ "${var.aws_account_id}" ]` and force the user to specify a account id

Reply:
Actually, TBH, I think the commit is out of scope for this PR anyway. I will do other one with the change and we discuss the topic there.

Reply:
removed